nautobot_device_lifecycle_mgmt/navigation.py

```diff
@@ -46,6 +46,31 @@
                 name="Software Lifecycle",
                 weight=100,
                 items=(
+                    NavMenuItem(
+                        link="plugins:nautobot_device_lifecycle_mgmt:softwarenotice_list",
+                        name="Software Notices",
+                        buttons=(
+                            NavMenuButton(
+                                link="plugins:nautobot_device_lifecycle_mgmt:softwarenotice_add",
+                                title="Add",
+                                icon_class="mdi mdi-plus-thick",
+                                button_class=ButtonColorChoices.GREEN,
+                                permissions=[
+                                    "nautobot_device_lifecycle_mgmt.add_softwarenotice",
+                                ],
+                            ),
+                            NavMenuButton(
+                                link="plugins:nautobot_device_lifecycle_mgmt:softwarenotice_import",
+                                title="Import",
+                                icon_class="mdi mdi-database-import-outline",
+                                button_class=ButtonColorChoices.BLUE,
+                                permissions=["nautobot_device_lifecycle_mgmt.add_softwarenotice"],
+                            ),
```

Comment:
1) Use `NavMenuAddButton` instead of a generic `NavMenuButton` for the "Add" button.
2) In current core versions we don't have an `Import` button in most nav menu entries any more.